features/step_definitions/custom_web_steps.rb

```diff
@@ -51,61 +36,43 @@ def matches_path?(url, path)
 # Then I should see "foo", "bar" and "baz"
 # Then I should see "foo", "bar", "baz" and "qux"
 # ...
-Then /^I should (see|not see) #{QUOTED_TWO_OR_MORE_PATTERN}$/ do |action, items|
+Then "I {should} see #{QUOTED_TWO_OR_MORE_PATTERN}" do |visible, items|
```

Comment:
So regular expressions actually work in a Cucumber expression? Also why is this not a transformation pattern?